calabash-cucumber/lib/calabash-cucumber/launcher.rb

```diff
@@ -221,12 +221,12 @@ def self.instruments?
       # @!visibility private
       def instruments?
         attached_to_gesture_performer? &&
-          gesture_performer.class.send(:name) == :instruments
+          @gesture_performer.class.send(:name) == :instruments
```

Comment:
@jmoody nope it was a non-blocking thing but I thought the missing instance variable was an unintentional change - my mistake.